tests/visualization_tests/matplotlib_tests/test_pareto_front.py

```diff
@@ -0,0 +1,223 @@
+        )
+
+    with pytest.raises(ValueError):
+        plot_pareto_front(
+            study=study,
+            target_names=["Foo", "Bar", "Baz"],
+            include_dominated_trials=include_dominated_trials,
+            axis_order=axis_order,
+        )
+
+    target_names = ["Foo", "Bar"]
+    figure = plot_pareto_front(
+        study=study,
+        target_names=target_names,
+        include_dominated_trials=include_dominated_trials,
+        axis_order=axis_order,
+    )
+    assert figure.has_data()
+
+
+@pytest.mark.parametrize("include_dominated_trials", [False, True])
+@pytest.mark.parametrize(
+    "axis_order", [None] + list(itertools.permutations(range(3), 3))  # type: ignore
+)
+def test_plot_pareto_front_3d(
+    include_dominated_trials: bool, axis_order: Optional[List[int]]
+) -> None:
+    # Test with no trial.
+    study = optuna.create_study(directions=["minimize", "minimize", "minimize"])
+    figure = plot_pareto_front(
+        study=study,
+        include_dominated_trials=include_dominated_trials,
+        axis_order=axis_order,
+    )
+
+    assert not figure.has_data()
+
+    # Test with three trials.
+    study.enqueue_trial({"x": 1, "y": 1, "z": 1})
+    study.enqueue_trial({"x": 1, "y": 0, "z": 1})
+    study.enqueue_trial({"x": 1, "y": 1, "z": 0})
+    study.optimize(
+        lambda t: [t.suggest_int("x", 0, 1), t.suggest_int("y", 0, 1), t.suggest_int("z", 0, 1)],
+        n_trials=3,
+    )
+
+    figure = plot_pareto_front(
+        study=study,
+        include_dominated_trials=include_dominated_trials,
+        axis_order=axis_order,
+    )
+    assert figure.has_data()
+
+    # Test with `target_names` argument.
+    with pytest.raises(ValueError):
+        plot_pareto_front(
+            study=study,
+            target_names=[],
+            include_dominated_trials=include_dominated_trials,
+            axis_order=axis_order,
+        )
+
+    with pytest.raises(ValueError):
+        plot_pareto_front(
+            study=study,
+            target_names=["Foo"],
+            include_dominated_trials=include_dominated_trials,
+            axis_order=axis_order,
+        )
+
+    with pytest.raises(ValueError):
+        plot_pareto_front(
+            study=study,
+            target_names=["Foo", "Bar"],
+            include_dominated_trials=include_dominated_trials,
+            axis_order=axis_order,
+        )
+
+    with pytest.raises(ValueError):
+        plot_pareto_front(
+            study=study,
+            target_names=["Foo", "Bar", "Baz", "Qux"],
+            include_dominated_trials=include_dominated_trials,
+            axis_order=axis_order,
+        )
+
+    target_names = ["Foo", "Bar", "Baz"]
+    figure = plot_pareto_front(study=study, target_names=target_names, axis_order=axis_order)
+
+    assert figure.has_data()
+
+
+@pytest.mark.parametrize("include_dominated_trials", [False, True])
+def test_plot_pareto_front_unsupported_dimensions(include_dominated_trials: bool) -> None:
```

Comment:
Since this test contains supported dimension cases: `n_objectives == 2` and `n_objectives == 3`, this test name is supposed to be inaccurate. I know the same test name exists in Plotly's test https://github.com/optuna/optuna/blob/62dc24574acd13d76ff1a4a94303a3302e5e3fbe/tests/visualization_tests/test_pareto_front.py#L212.

Personally, I suggest changing this test name to `test_plot_pareto_front_dimensions`, but I may miss some context of this name, so I would like to hear the opinions from the core-team members.